ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx

```diff
@@ -1,11 +1,13 @@
 import { format } from "date-fns";
 import { ListItem } from "native-base";
 import React from "react";
-import { View, Image, StyleSheet } from "react-native";
+import { Image, StyleSheet, View } from "react-native";
 import { OperationTypeEnum as IbanOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/IbanOperationDTO";
 import { OperationTypeEnum as OnboardingOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/OnboardingOperationDTO";
 import { OperationListDTO } from "../../../../../../definitions/idpay/timeline/OperationListDTO";
 import { OperationTypeEnum as RefundOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/RefundOperationDTO";
+import { OperationTypeEnum } from "../../../../../../definitions/idpay/timeline/RejectedInstrumentOperationDTO";
```

Comment:
```suggestion
import { OperationTypeEnum as  RejectedInstrumentOperationTypeEnum } from "../../../../../../definitions/idpay/timeline/RejectedInstrumentOperationDTO";
```

Maybe a bit too long but helps keeping code clear imho.

Reply:
@forrest57 Can you fix this?